components/Report/models.rst

```diff
@@ -0,0 +1,73 @@
+Models
+======
+
+The Report
+-----------
+
+Report is the main model in SyliusReportComponent. This simple class represents every unique report in the system.
+The default interface contains the following attributes with appropriate setters and getters.
+
++---------------------------+----------------------------------------------------+
+| Attribute                 | Description                                        |
++===========================+====================================================+
+| id                        | Unique id of the report                            |
++---------------------------+----------------------------------------------------+
+| code                      | Unique code of the report                          |
++---------------------------+----------------------------------------------------+
+| name                      | Name of the report                                 |
++---------------------------+----------------------------------------------------+
+| description               | Description of your gorgeous report                |
++---------------------------+----------------------------------------------------+
+| renderer                  | Name of the renderer that visualize report data    |
++---------------------------+----------------------------------------------------+
+| rendererConfiguration     | Configuration of used renderer                     |
++---------------------------+----------------------------------------------------+
+| dataFetcher               | Name of the dataFetcher that provides report data  |
++---------------------------+----------------------------------------------------+
+| dataFetcherConfiguration  | Configuration of used data fetcher                 |
++---------------------------+----------------------------------------------------+
+
+.. note::
+
+    This model implements ``ReportInterface``
+
+
+The Data
+-----------
+
+Data is supporting model in SyliusReportBundle. That contains data provided by data fetcher and labels used to clarify report display.
```

Comment:
Component doesn't know anything about SyliusReportBundle. Data model represents report data in a uniform form.